src/main/java/org/jboss/ejb/protocol/remote/RemoteServer.java

```diff
@@ -539,7 +546,7 @@ public void writeResponse(final Object response) {
             }
         }
 
-        public void writeException(final Exception response) {
+        public void writeThrowable(final Throwable response) {
```

Comment:
We don't ever want to write Errors or anything like that; generally the EJB protocol and spec deal in Exceptions only.  Was there some reason this had to be widened?